cpu/esp32/Makefile.features

```diff
@@ -6,9 +6,12 @@ include $(RIOTCPU)/esp_common/Makefile.features
 
 FEATURES_PROVIDED += arch_esp32
 FEATURES_PROVIDED += esp_wifi_enterprise
-FEATURES_PROVIDED += esp_hw_counter
 FEATURES_PROVIDED += puf_sram
 
+ifneq (,$(filter esp32,$(CPU_FAM)))
```

Comment:
Not

```suggestion
ifeq (xtensa,$(CPU_ARCH)))
```

?

Reply:
I'm going to change it, even though it will lead to conflicts with the subsequent PRs for ESP32-S2 and ESP32-S3. Since the PR is so small, I guess that I can squash directly.

Reply:
If we define it as a feature dependent on `CPU_ARCH`, it should probably be changed in Kconfig as well.

Reply:
Oops, had the same idea before I reloaded.

Reply:
Hm, in Kconfig it is a bit more complicated, we don't have any `CPU_ARCH_XTENSA_ESP32` or something like that for identifying Xtensa-based ESP32s. We can't use `CPU_ARCH_XTENSA` to select the feature since ESP8266 doesn't support it. Maybe, it is better to let define the family this feature in Kconfig. The result should be the same.

Reply:
@benpicco Commit 6b3fe4aa70cbf583a514fcbdd6a5f9efe601811c could be a solution by defining one more `CPU_ARCH_ESP` symbol. What do you think?